src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-emr-monitoring-integration.mdx

```diff
@@ -67,7 +67,7 @@ This integration collects the following Amazon EMR data. For use cases and addit
       <td>
         Indicates that a cluster is no longer performing work, but is still alive and accruing charges. It is set to 1 if no tasks are running and no jobs are running, and set to 0 otherwise.
 
-        This value is checked at five-minute intervals, and a value of 1 indicates only that the cluster was idle when checked, not that it was idle for the entire five minutes. <DNT>**Recommendation:**</DNT> To avoid false positives, raise an alerting threshold when this value has been 1 for more than one consecutive five-minute check. For example, raise an alert on this value if it has been 1 for thirty minutes or longer.
+        This value is checked at five-minute intervals, and a value of 1 indicates only that the cluster was idle when checked, not that it was idle for the entire five minutes. Recommendation:To avoid false positives, raise an alerting threshold when this value has been 1 for more than one consecutive five-minute check. For example, raise an alert on this value if it has been 1 for thirty minutes or longer.
```

Comment:
```suggestion
        This value is checked at five-minute intervals, and a value of 1 indicates only that the cluster was idle when checked, not that it was idle for the entire five minutes. Recommendation: To avoid false positives, raise an alerting threshold when this value has been 1 for more than one consecutive five-minute check. For example, raise an alert on this value if it has been 1 for thirty minutes or longer.
```